### PR TITLE
Integrated Cloudinary images into member page

### PR DESCRIPTION
**Pull request by prakashchoudhary07**

### CODE REVIEW

File: src/helper-functions/urls.js [hunk 1]
```diff
@@ -1,13 +1,22 @@
 const baseURL = `https://api.realdevsquad.com`;
 const imgBaseURL = `https://raw.githubusercontent.com/Real-Dev-Squad/website-static/main`;
 const getMembersURL = `${baseURL}/members`;
+const cloudinaryImageUrl = `https://res.cloudinary.com/realdevsquad/image/upload`;
```
Comment (sumitd94) [inline src/helper-functions/urls.js:4]: Can we please move this also to constants?

Comment (prakashchoudhary07) [reply src/helper-functions/urls.js:4]: This file contains all the url's, I don't get the reason for moving this to another constant file and using it here again

Comment (sumitd94) [reply src/helper-functions/urls.js:4]: Ah! sorry, my bad.

**Review by sumitd94 — APPROVED** [review]

File: src/pages/index.js [hunk 2]
```diff
@@ -43,9 +48,12 @@ export async function getServerSideProps(context) {
     const { members } = await res.json();
 
     for (const memberData of members) {
+      const img_url = memberData.picture
+        ? getCloudinaryImgURL(memberData.picture.publicId, `w_${MAX_WIDTH}`)
+        : getImgURL(memberData.username, 'img.png');
```
Comment (sumitd94) [inline src/pages/index.js:53]: Can we add this behind the feature flag? meaning this feature should only work if `dev=true`

**Review by harshith-venkatesh — APPROVED** [review]

**Review by sumitd94 — APPROVED** [review]

**Pull request merged**